src/Exception/DriverException.php

```diff
@@ -4,23 +4,41 @@
 
 use Doctrine\DBAL\Driver\Exception as TheDriverException;
 use Doctrine\DBAL\Exception;
+use Doctrine\DBAL\Query;
 
 use function assert;
 
 /**
  * Base class for all errors detected in the driver.
  *
+ * @internal
```

Comment:
The class itself shouldn't be made internal. Consumers of the DBAL API may catch it. Only the constructor may be made internal.

Reply:
Oops, my bad. That was what I intended.